docs: add warning for Git LFS (#255)
instill-ai/instill.tech: docs/import-models/github.mdx

Assume Git LFS is [installed](https://git-lfs.github.com/), this guideline publishes model files in a repository on GitHub.

> :::info{type=warning}

Reviewer: ```suggestion
:::info{type=info}
```

> Importing models from GitHub using Git LFS is not recommended unless you're with a private GitHub Enterprise account.

Reviewer: ```suggestion
In general, importing models via this approach is not recommended. GitHub has limited quotas of storage and bandwidth for Git LFS files, and the usage will count against the repository owner's quotas. Unless you purchase more quotas.

Instead, consider using GitHub DVC or ArtiVC approaches.
```

Author: ```suggestion
In general, importing models via this approach is not recommended. GitHub has limited quotas of storage and bandwidth for Git LFS files, and the usage will count against the repository owner's quotas leading you to purchase more.

Instead, consider using GitHub DVC or ArtiVC approaches.
```

> GitHub has a limit of [2 GB per repository](https://docs.github.com/en/repositories/working-with-files/managing-large-files/about-git-large-file-storage) for using Git LFS

Reviewer: ```suggestion
```

> and bandwidth and storage usage will count against the repository owner's quotas.

Reviewer: ```suggestion
```

- [Create a storage bucket](https://cloud.google.com/storage/docs/creating-buckets) before adding DVC remote
- Make sure to run `gcloud auth application-default login` or other ways to authenticate and access GCS.

>   :::

Reviewer: ```suggestion
:::
```
